ark/utils/io_utils.py

```diff
@@ -72,18 +72,44 @@ def list_files(dir_name, substrs=None):
     return matches
 
 
-def extract_delimited_names(files, delimiter='_', delimiter_optional=True):
-    """Create a matched-index list of fov names from a list of files/folders
+def remove_file_extensions(files):
+    """Removes file extensions and drops preceding path from a list of files
 
-    Extracts a delimited prefix for every file in a given list of files
+    Args:
+        files (list):
+            List of files to remove file extensions from.
+            Any element that doesn't have an extension is left unchanged
+
+    Returns:
+        list:
+            List of files without file extensions
+    """
+
+    # make sure we don't try to split on an undefined list of files
+    if files is None:
+        return
+
+    # only get the file name and not the directory path leading up to it
+    names = [os.path.split(name)[1] for name in files]
 
-    Examples:
-         - 'fov2_restofthefilename.tiff' becomes 'fov2'
-         - 'fov1.tiff' becomes 'fov1'
+    # remove anything past and including the first '.' in each entry
+    names = [name.split('.')[0] for name in names]
```

Comment:
This will fail if the filename has periods in it, which while not ideal, does happen. You can use os.path.splitext

Reply:
Got it. By the way, this method will fail if the user passes in a file with more than 1 `.` in the extension (ex. `my_sample_tar.tar.gz`). Do we expect this case to pop up often, or can we safely assume that we won't be seeing it?

Reply:
Agreed, if their image data is still in a tarball nothing is going to work anyway. Maybe add a warning message that gets raised if the cleaned up version of the filename still has a period in it.